README.md

```diff
@@ -329,6 +329,44 @@ services.AddSwaggerForOcelot(Configuration,
   });
 ```
 
+### Control downstream to swagger api
```

Comment:
Please can you move this section under `### If none of this is enough` and rename it to `## Control downstream to swagger api`. Because this main section is about **Documentation of Ocelot Aggregates**.

Thanks.

Reply:
updated readme